console/console-init/ui/src/modules/iot-project-detail/DetailPage.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { Grid, GridItem } from "@patternfly/react-core";
+import {
+  GeneralInfo,
+  DeviceRegistationManagement,
+  AccessCredentials,
+  IMessagingObject
+} from ".";
+import { action } from "@storybook/addon-actions";
+import { IAdapterConfig, IAdapter } from "components/AdapterList";
+import { Protocols } from "constant";
+
+export default function DetailPage() {
+  const eventAddresses: Array<string> = ["event_address", "event_address1"];
+  const telemetryAddresses: Array<string> = ["telemetry_address"];
+  const commandAddresses: Array<string> = ["command_address"];
+
+  const messaging: IMessagingObject = {
```

Comment:
Please rename IMessagintObject to IIoTMessagingObject as it create confusion in between IOT and Messaging Projects